lib/Frontend/ArgsToFrontendOutputsConverter.cpp

```diff
@@ -282,9 +282,7 @@ SupplementaryOutputPathsComputer::computeOutputPaths() const {
   else {
     // Multi-threaded WMO is the exception
     assert(OutputFiles.size() == InputsAndOutputs.inputCount() &&
-                   pathsFromUser->size() == InputsAndOutputs.hasInputs()
-               ? 1
-               : 0);
+                   pathsFromUser->size() == (InputsAndOutputs.hasInputs() ? 1 : 0));
```

Comment:
Could you `clang-format` this?

Reply:
output from `clang-format:
```
           pathsFromUser->size() == (InputsAndOutputs.hasInputs() ? 1 : 0));
```

Reply:
> Yes, so the column of both && operands should match.

No, they shouldn't as strings before && don't match.
I've changed indentation to format assert exactly like one above it.

Reply:
> I've changed indentation to format assert exactly like one above it.

The indentation is now correct, thanks. I think you misunderstood my point though: I was reasserting, not contradicting the formatter output, that both `&&` operands (`OutputFiles.size() == InputsAndOutputs.inputCount()` and `pathsFromUser->size() == (InputsAndOutputs.hasInputs() ? 1 : 0)`) should be aligned by column. (as they are now)